src/rules.js

```diff
@@ -1903,6 +1903,80 @@ module.exports = [
     },
     /**
      ==================================================================
+     TRANSITION
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        id: 'transition',
+        name: 'Transition',
+        prefix: '.Trs-',
+        properties: ['transition'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true
```

Comment:
We could have something like `Trs-all_2s`isn't?

We need to make sure Atomizer replaces the `_` with a space though.
I'll open an issue to make sure we have that on our radar if this does not work like this yet.